.github/workflows/java-microservice-ci-default.yml

```diff
@@ -360,7 +360,35 @@ jobs:
           name: docker-sbom-${{ inputs.project_name }}-service.json
           path: ${{github.workspace}}/target/docker-sbom-${{ inputs.project_name }}-service.json
 
-      # Dependencies checking
+      # Clair checking
+      - name: Clair scan docker db
+        uses: quay/clair-action@main
```

Comment:
I prefer fix version than main

Reply:
Corrected but documentation is using main, it is maybe better while the action is still in early development